frontend/public/components/monitoring.tsx

```diff
@@ -915,25 +916,21 @@ export class MonitoringUI extends React.Component<null, null> {
       store.dispatch(UIActions.monitoringErrored('alerts', new Error('prometheusBaseURL not set')));
     }
 
-    const {alertManagerBaseURL} = (window as any).SERVER_FLAGS;
-    if (!alertManagerBaseURL) {
-      const e = new Error('alertManagerBaseURL not set');
-      store.dispatch(UIActions.monitoringErrored('alerts', e));
```

Comment:
This line should have been removed as part of https://github.com/openshift/console/pull/872/commits/c4c82d2f45fd01799d48185a52b9e655248ba0ee.